README.md

```diff
@@ -3,38 +3,55 @@
 [![GoReportCard](http://goreportcard.com/badge/elastic/beats)](http://goreportcard.com/report/elastic/beats)
 [![codecov.io](https://codecov.io/github/elastic/beats/coverage.svg?branch=master)](https://codecov.io/github/elastic/beats?branch=master)
 
-# Beats - Lightweight shippers for Elasticsearch & Logstash
+# Beats - The Lightweight Shippers of the Elastic Stack
 
-The [Beats](https://www.elastic.co/products/beats) are lightweight processes,
-written in Go, that you install on your servers to capture all sorts of
-operational data like logs, operating system metrics or network packet data,
-and to send it to Elasticsearch, either directly or via Logstash, so it can be
-visualized with Kibana.
+The [Beats](https://www.elastic.co/products/beats) are lightweight data
```

Comment:
Yeah, that was one ridiculously long sentence :-). I changed it mostly to your version.